tools/build_third_party.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+# coding=utf-8
+#
+# build_third_party.py
+# --------------------
+# This script updates the third party dependencies of deno.
+#
+# - Get Depot Tools and make sure it's in your path.
```

Comment:
It might be nice to print an error when depot tools aren't available/found

Reply:
ditto for the yarn dep

Reply:
Thanks for the comment! I agree. I'll try it.

Reply:
Actually I prefer not adding extra code for those checks. `run()` will fail with a stack trace that can diagnose that issue. This script is not user facing.